checkpoints/checkpoint2/checkpoint2/Portfolio/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
Your site has some support for smaller screens but does not really handle the transition well from wide screens to smaller screens.